docs/user/06-conditions.md

```diff
@@ -0,0 +1,37 @@
+# Telemetry CR conditions
+
+This section describes the possible states of the Telemetry CR. There are conditions type `LogComponentsHealthy`, `TraceComponentsHealthy` and `MetricComponentsHealthy` representing state of each of the subcomponents. The state of the Telemetry CR is derived from combined state of all the subcomponents.
+
+## Log Components State
```

Comment:
For example, drop the column "Condition Type" and instead write:

```suggestion
## Log Components State

The state of the log components is determined by the status of the condition type `LogComponentsHealthy`:
```

Reply:
Changed it. Also rewrote the last paragraph, the table there was not mathematically correct (not all options were listed). Changed it to a plane text.